projects/1.16/assets/tinkers-construct/tconstruct/book/materials_and_you/zh_cn/modifiers/abilities.json

```
@@ -1,16 +1,15 @@
 {
-  "title": "能力",
-  "text": [
-    {
-      "text": "能力是一种非常强大的工具强化，它甚至可以彻底改变一个工具。类似于精准采集或者是自动熔炼这样的强化，值得注意的是，他们都改变了采掘方块的效果。而双持则使得工具更容易操作。"
-    },
-    {
-      "text": "能力需要装载进能力槽中。几乎所有的工具初始都会携带一个能力槽，且最多只能装载一个能力，所以最好在选择强化时明智一些。",
-      "paragraph": true
-    },
-    {
-      "text": "Thruul·M'Gon的入门熔炼术有着一整套用于工具的可用强化列表，当然，匠魂大宝典（Selena著）也是有的。",
-      "paragraph": true
-    }
-  ]
-}
+    "title": "能力",
+    "text": [{
+            "text": "能力是一种非常强大的工具强化，它甚至可以彻底改变一个工具。诸如精准采集和自动冶炼这样的强化非常瞩目，它们彻底改变了采掘矿物的效果，而双持强化则让工具更便于操作。"
+        },
+        {
+            "text": "能力需要装载进能力槽中。几乎所有的工具初始都会携带一个能力槽，且最多只能装载一个能力，所以最好在选择强化时明智一些。",
```

```suggestion
            "text": "能力需要加持进能力槽中。几乎所有的工具初始都会携带一个能力槽，且最多只能加持一个能力，所以最好在选择强化时明智一些。",
```
这里本来库里的用词是“加持”
有必要就统一一下吧